spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -110,6 +115,31 @@
     }
   }
 
+  /** Class to contain venue sample data. */
+  static class Venue {
+
+    final long venueId;
```

Comment:
If the class is Venue, then why mention in on each member?  Your going to type it a lot?

Reply:
Leaving as is for now.